quickget

```diff
@@ -697,12 +697,18 @@ function releases_crunchbang++() {
 }
 
 function releases_debian() {
+    local MAJ=""
     local NEW=""
     local OLD=""
     NEW=$(web_pipe "https://cdimage.debian.org/debian-cd/" | grep '\.[0-9]/' | cut -d'>' -f 9 | cut -d'/' -f 1)
-    OLD=$(web_pipe "https://cdimage.debian.org/cdimage/archive/" | grep -e '>[1-9][0-9]\.' | grep -v 'live' | grep -v NEVER | cut -d'>' -f 9 | cut -d'/' -f 1 | tac | head -20)
-    #shellcheck disable=SC2086
-    echo ${NEW} ${OLD}
+    echo -n "${NEW}"
+    MAJ=$(echo "${NEW}" | cut -d'.' -f 1)
+    for i in {1..2}; do
+        CUR=$((MAJ - i))
+        OLD=$(web_pipe "https://cdimage.debian.org/cdimage/archive/" | grep folder | grep -v 'live' | grep -v NEVER | cut -d'"' -f 6 | grep ^"${CUR}" | tail -n 1 | tr -d '/')
+        echo -n " ${OLD}"
+    done
```

Comment:
Why are we making a request to the same URL twice? Instead, shouldn't you store that as a variable and then filter through the data in a loop.

```suggestion
    local OLD_DATA
    OLD_DATA="$(web_pipe "https://cdimage.debian.org/cdimage/archive/" | grep folder | grep -v 'live' | grep -v NEVER | cut -d'"' -f 6)"
    for i in {1..2}; do
        CUR=$((MAJ - i))
        OLD=$(grep ^"${CUR}"  <<< "${OLD_DATA}" | tail -n 1 | tr -d '/')
        echo -n " ${OLD}"
    done
```

Reply:
Good idea. I've pushed the change.